src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java

```diff
@@ -354,4 +361,37 @@ public void describeTo(final Description description) {
         return CoreMatchers.both(Matchers.greaterThanOrEqualTo(400)).and(Matchers.lessThan(500));
     }
 
+    protected void confirmPresenceOfLinkValue(final String linkValue, final Response response) {
+        final Link link = Link.valueOf(linkValue);
+        final String relType = link.getRel();
```

Comment:
The `relType` variable is never used. I would suggest either removing it, or using it in line:367.